src/tracker.js

```diff
@@ -3,10 +3,10 @@ import { makePromise } from './utils'
 const TIMEOUT = new Error('Tracker timeout')
 
 class Tracker {
-  constructor(depthLimit, timeLimit, warnOnMaxDepth, logger) {
+  constructor(renderLimit, timeLimit, warnOnMaxDepth, logger) {
```

Comment:
The amount of constructor parameters is getting too large. It might be a good idea to change this to `{renderLimit ...etc }` instead